e2e/knative/files/setup.sh

```diff
@@ -77,7 +77,7 @@ EVENTING_CORE="https://github.com/knative/eventing/releases/download/${EVENTING_
 IN_MEMORY_CHANNEL="https://github.com/knative/eventing/releases/download/${EVENTING_VERSION}/in-memory-channel.yaml"
 CHANNEL_BROKER="https://github.com/knative/eventing/releases/download/${EVENTING_VERSION}/mt-channel-broker.yaml"
 
-KNATIVE_TEMP=$(mktemp -d knative-XXXX)
+KNATIVE_TEMP=$(mkdir -p ./build/_output && mktemp -d ./build/_output/knative-XXXX)
```

Comment:
Watch out. The content of build is going to be bundled into the docker container when `make images`. I suggest to use a different folder for the scope. Also, is `mktemp` cross OS compatible? not sure if it will complain on mac users.

Reply:
The content of specific `./build` subdirs are copied to the image, not all of `./build/*`. 
I tested this on Linux and MacOS (otherwise it would not work me either)

Reply:
True. I've checked the Dockerfile and we pick certain subdirectories only.